index.js

```diff
@@ -1,35 +1,42 @@
-/* jshint node: true */
+/* eslint-env node */
 'use strict';
 
 var path = require('path');
-var Funnel = require('broccoli-funnel');
 var MergeTrees = require('broccoli-merge-trees');
+var Funnel = require('broccoli-funnel');
+var map = require('broccoli-stew').map;
 
 module.exports = {
   name: 'ember-cli-dropzonejs',
-  included() {
-    this._super.included.apply(this, arguments);
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      this.import('vendor/dropzone.min.js');
-      this.import('vendor/dropzone.min.css');
-    }
-  },
+
   treeForVendor(vendorTree) {
-    if (!process.env.EMBER_CLI_FASTBOOT) {
-      let trees = [];
-      let dropzoneTree = new Funnel(
-        path.join(this.project.root, 'node_modules', 'dropzone/dist/min'),
-        {
-          files: ['dropzone.min.js', 'dropzone.min.css']
-        }
-      );
-
-      trees.push(dropzoneTree);
-
-      if (vendorTree) {
-        trees.push(vendorTree);
+    var dropzoneJs = new Funnel(
+      path.join(this.project.root, 'node_modules', 'dropzone/dist/min'),
+      { files: ['dropzone.min.js'] }
+    );
+
+    dropzoneJs = map(
+      dropzoneJs,
+      content => `if (typeof FastBoot === 'undefined') { ${content} }`
+    );
+
+    return new MergeTrees(vendorTree, dropzoneJs);
+  },
+
+  treeForStyles(styleTree) {
+    var dropzoneCss = new Funnel(
+      path.join(this.project.root, 'node_modules', 'dropzone/dist/min'),
+      {
+        files: ['dropzone.min.css'],
+        destDir: 'app/styles'
       }
-      return new MergeTrees(trees);
-    }
+    );
+
+    return new MergeTrees(styleTree, dropzoneCss);
```

Comment:
Seems trying to merge with a null tree throws an error `Expected Broccoli node, got undefined for inputNodes[0]`